src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Json/DataContractJsonSerializer.cs

```diff
@@ -328,10 +328,12 @@ public void WriteObject(Stream stream, object graph)
 #endif
         }
 
+#if !MERGE_DCJS
```

Comment:
This method was used by both NetNative and NetCore. Does this mean NetCore does not need this method?